src/common/router/index.ts

```diff
@@ -86,6 +86,16 @@ const routes: Readonly<RouteRecordRaw[]> = [
       () => import(/* webpackChunkName: "pegin-success" */ '../../pegin/views/Success.vue'),
     beforeEnter: [checkFromRoute],
   },
+  {
+    path: '/:type/success/tx/:txId',
```

Comment:
Just a general comment. Do we no longer want to keep the wallet or if it is a flyover operation within the URL? Why?